nanofi/include/sitetosite/CRawSocketProtocol.h

```diff
@@ -0,0 +1,219 @@
+/**
+ * @file RawSiteToSiteClient.h
```

Comment:
Not a big deal but if you have another commit might want to resolve some of these.

Reply:
Done, thanks